libs/sdk-core/src/swap.rs

```diff
@@ -386,12 +391,14 @@ impl BTCReceiveSwap {
         if swap_info.bolt11.is_none() {
             let invoice = self
                 .payment_receiver
-                .receive_payment(
-                    swap_info.confirmed_sats,
-                    String::from("Bitcoin Transfer"),
-                    Some(swap_info.preimage),
-                )
-                .await?;
+                .receive_payment(ReceivePaymentRequestData {
+                    amount_sats: swap_info.confirmed_sats,
+                    description: String::from("Bitcoin Transfer"),
+                    preimage: Some(swap_info.preimage),
+                    opening_fee_params: None,
```

Comment:
Above comment is irrelevant now.

What we need to do here:
If `swap_info` doesn't contain `opening_fee_params`, get a new `opening_fee_params` from the LSP that's valid for 48 hours and pass that to `receive_payment`. That would be the new 'backward compatible flow'.

Reply:
Already covovered in https://github.com/breez/breez-sdk/pull/313#discussion_r1272072796

Reply:
As discussed, `receive_payment` already defaults internally to looking up the cheapest fee if it's called with `None` as fee arg. Since redeeming a swap happens quasi-instantly (as fast as settling a LN invoice), the cheapest dynamic fee is good enough as feedback.